src/c++/perf_analyzer/concurrency_worker.cc

```diff
@@ -177,13 +177,16 @@ ConcurrencyWorker::WaitForResponses()
     {
       // If async, then wait for signal from callback.
       std::unique_lock<std::mutex> lk(cb_mtx_);
+      thread_stat_->idle_timer.Start();
+      auto start = std::chrono::steady_clock::now();
```

Comment:
I don't see start being used anywhere? What is the purpose of this line?

Reply:
removed